testsuite/.rubocop_todo.yml

```diff
@@ -48,6 +48,7 @@ Layout/ClosingHeredocIndentation:
 Layout/EmptyLineAfterGuardClause:
   Exclude:
     - 'features/step_definitions/api_common.rb'
+    - 'features/step_definitions/cobbler_steps.rb'
```

Comment:
Let's try to refactor the code causing this warning, so we don't need to include one more excluded entry.